optuna/trial/_frozen.py

```diff
@@ -589,7 +589,7 @@ def create_trial(
     user_attrs = user_attrs or {}
     system_attrs = system_attrs or {}
     intermediate_values = intermediate_values or {}
-    state = state or TrialState.COMPLETE
+    state = state
```

Comment:
This line is not needed any more.

Reply:
Thanks for your advice. I removed the line. PTAL.